src/editor/index.js

```diff
@@ -29,17 +31,32 @@ class PopupSidebar extends Component {
 		const {
 			dismiss_text,
 			frequency,
+			id,
 			onMetaFieldChange,
+			onSitewideDefaultChange,
 			overlay_opacity,
 			overlay_color,
 			placement,
+			newspack_popups_is_sitewide_default,
 			trigger_scroll_progress,
 			trigger_delay,
 			trigger_type,
 			utm_suppression,
 		} = this.props;
 		return (
 			<Fragment>
+				<CheckboxControl
+					label={ __( 'Sitewide Default', 'newspack-popups' ) }
+					checked={ newspack_popups_is_sitewide_default }
+					onChange={ value => {
+						const params = {
+							path: '/newspack-popups/v1/sitewide_default/' + id,
+							method: value ? 'POST' : 'DELETE',
+						};
+						onSitewideDefaultChange( value );
+						apiFetch( params );
+					} }
+				/>
```

Comment:
It feels a inconsistent that this input updates the popup settings _immediately_, while other inputs just change the "edited" state of the popup (awaiting publish). This will also result in a following scenario being possible:
1. user adds new popup
1. they fill in title, content, check "Sitewide Default", and publish
1. the popup is **not** Sitewide Default though, because at the time of checking the checkbox, the post was not existing yet

Maybe the option update logic could be handled [on post save (`didPostSaveRequestSucceed`)](https://developer.wordpress.org/block-editor/data/data-core-editor/#didPostSaveRequestSucceed)?